libblkid/meson.build

```diff
@@ -141,7 +141,10 @@ lib_blkid = both_libraries(
 
 lib_blkid_static = lib_blkid.get_static_lib()
 
-pkgconfig.generate(lib_blkid,
-                   description : 'Block device id library',
-                   subdirs : 'blkid',
-                   version : pc_version)
+if build_libblkid
+  pkgconfig.generate(lib_blkid,
+                     description : 'Block device id library',
+                     subdirs : 'blkid',
+                     version : pc_version)
+  install_headers(join_paths(meson.current_build_dir(), 'blkid.h'), subdir : 'blkid')
```

Comment:
In the above configure_file which creates this, you should use `install: build_libblkid, install_dir: join_paths(get_option('includedir'), 'blkid')`.